### PR TITLE
Do release builds using static launchpad recipes

### PR DESCRIPTION
**Pull request by alfonsosanchezbeato**
So Pro can be enabled in the static recipes.

### CODE REVIEW

**Comment by alfonsosanchezbeato** [comment]
Draft until we get pro permissions for ~snappy-hwe-team.

File: workflows/trigger-lp-build.py [hunk 1]
```diff
@@ -100,7 +93,22 @@ def main(argv):
...
 
     snap = None
-    if ephemeral_build:
+    ephemeral_build = False
+    if args['git_repo_branch'] is not None and \
+       args['git_repo_branch'].startswith('snap-'):
+        # We remove the temporal branch suffix
+        rec_suffix = re.sub('_.*', '', args['git_repo_branch'])
+        build_name = "%s-%s" % (args['snap'], rec_suffix)
+        print('Getting snap recipe {} from {} team'.format(build_name, team))
```
Comment (bboozzoo) [inline workflows/trigger-lp-build.py:102]: Perhaps we can gradually start switching the new code to use `f"..."` strings. They're available from 3.6, so hopefully supported everywhere where we could run these helpers.

File: workflows/snap-release.sh [hunk 2]
```diff
@@ -380,4 +381,5 @@ if [ $# -ne 2 ]; then
     exit 1
 fi
 NEXT_VERSION=${NEXT_VERSION:-}
+BUILD_BRANCH="$1"_$CI_ID
```
Comment (bboozzoo) [inline workflows/snap-release.sh]:
```suggestion
BUILD_BRANCH="${1}_${CI_ID}"
```

**Review by bboozzoo — APPROVED** [review]
LGTM

**Review by Meulengracht — APPROVED** [review]
lgtm

**Pull request merged**